Makefile

```diff
@@ -45,7 +45,11 @@ install: gopath
 	install -D -m 00644 yum.conf.in $(DESTDIR)/usr/share/defaults/mixer/yum.conf.in
 
 check: gopath
-	go test ${GO_PACKAGE_PREFIX}/...
+	go test -cover ${GO_PACKAGE_PREFIX}/...
+
+checkcoverage: gopath
+	test ${PKG}
+	go test -cover ${GO_PACKAGE_PREFIX}/${PKG} -coverprofile=coverage.out
```

Comment:
Since checkcoverage seems to be focused on developer use (and not the CI, that uses check), I wonder if you could also call the coverage visualization tool, adding this line to the rules

    go tool cover -html=coverage.out